cypress/e2e/shared/deepLinks.test.ts

```diff
@@ -12,7 +12,6 @@ describe('Deep linking', () => {
   // so you'll probably need to follow-up with the docs and/or marketing teams.
   it('should be redirected to the approprate page from a shortened link', () => {
     cy.get('@org').then((org: Organization) => {
-      cy.wait(1000)
```

Comment:
this `wait` was part of the feature flagging code, which waited after flipping the feature flag, but (erroneously) wasn't removed when the feature flag was removed